### PR TITLE
Update book to include typable command remapping

### PR DESCRIPTION
**Pull request by Nehliin**
cc @archseer

### CODE REVIEW

**Review by dead10ck — APPROVED** [review]

File: book/src/remapping.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ this:
 ```toml
 # At most one section each of 'keys.normal', 'keys.insert' and 'keys.select'
 [keys.normal]
+C-s = ":w" # Maps the Control-s to the typable command :w which is an alias for :write (save file)
```
Comment (archseer) [inline book/src/remapping.md:14]: Let's add another example that uses arguments, for example `:open ~/.config/helix/config.toml`

File: book/src/remapping.md [hunk 2]
```diff
@@ -53,4 +56,4 @@ Control, Shift and Alt modifiers are encoded respectively with the prefixes
 Keys can be disabled by binding them to the `no_op` command.
 
 Commands can be found at [Keymap](https://docs.helix-editor.com/keymap.html) Commands.
```
Comment (python128) [inline book/src/remapping.md:58]: Here, we should also add the link for TypeableCommands.

**Review by python128 — CHANGES_REQUESTED** [review]

File: book/src/remapping.md (hunk 2 above)
Comment (python128) [reply book/src/remapping.md:58]: Such as:
```markdown
Typeable Commands can be found at [Commands](https://docs.helix-editor.com/commands.html)
```

**Comment by Deeprai32037i** [comment]
Fonce

File: book/src/remapping.md (hunk 2 above)
Comment (Deeprai32037i) [reply book/src/remapping.md:58]: Gâcher

**Pull request merged**